### PR TITLE
Task swe 1403 create shades components

### PR DESCRIPTION
**Pull request by olenashevchuk**

### CODE REVIEW

File: src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Box, Text, Title } from 'components'
+
+import ShadeAdvancedViewStyled from './ShadeAdvancedView.styled'
+
+const ShadeAdvancedView = (props) => {
+  const { color, name, textColor, hex, rgb } = props
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js:6]: propTypes

File: src/app/domains/Color/components/ShadeSimpleView/ShadeSimpleView.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import ShadeSimpleViewStyled from './ShadeSimpleView.styled'
+import { Text } from 'components'
+
+const ShadeSimpleView = (props) => {
+  const { color, textColor, percentages } = props
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ShadeSimpleView/ShadeSimpleView.js:5]: propTypes

File: src/app/domains/Color/components/ShadesList/ShadesList.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { ShadeAdvancedView, ShadeSimpleView } from 'domains/Color/components'
+
+import ShadesListStyled from './ShadesList.styled'
+
+const ShadesList = (props) => {
+  const { isReverse, shades, mainColorData } = props
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ShadesList/ShadesList.js:6]: propTypes

File: src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+import PropTypes from 'prop-types'
+import ShadeAdvancedViewStyled from './ShadeAdvancedView.styled'
+
+/**
+ * It's a function that returns a styled component that contains a color's name, hef and rgb value.
+ * It is used for show advanced shade view
+ */
+const ShadeAdvancedView = (props) => {
+  const { color, name, textColor, hex, rgb, isReverse } = props
```
Comment (BogdanovJn) [inline src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js:11]: I feel some confusion when I see "color" and "textColor". May be let's swap "color" to smth. like "shade"/"value"/"bg"? @aleksandrgolyk your thoughts?

File: src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+  return (
+    <ShadeAdvancedViewStyled color={color} isReverse={isReverse}>
+      <Box mb={2}>
+        <Title variant="heading6" color={textColor}>
```
Comment (BogdanovJn) [inline src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js:16]: "heading6" - too long, "h6" will be much more better

Comment (olenashevchuk) [reply src/app/domains/Color/components/ShadeAdvancedView/ShadeAdvancedView.js:16]: okay, Olexandr Voichyshyn will fix it globally in next PR

**Comment by olenashevchuk** [comment]
Screenshot from 2022-07-26 16-29-17

**Review by BogdanovJn — APPROVED** [review]

**Pull request merged**